sdk/src/main/java/com/silanis/esl/sdk/builder/DocumentPackageSettingsBuilder.java

```diff
@@ -330,6 +332,38 @@ public DocumentPackageSettingsBuilder withHandOverLinkText(String text) {
         return this;
     }
 
+    /**
+     * Whether to use time based Expiry of package. If enabled, set the number
+     * of days #withRemainingDays since pacakge creation date the package should expire.
+     * <p>
+     * DEFAULT: DISABLED
+     * <p>
+     *
+     * @return This
+     * @see #enableTimeBasedExpiry()
+     */
+
+    public DocumentPackageSettingsBuilder enableTimeBasedExpiry() {
+        this.enableTimeBasedExpiry = true;
+        return this;
+    }
```

Comment:
If you added the enable method, add the disable method as well. Later, the default value can be changed at any time.

Reply:
done